src/DynamoPackagesUI/DynamoPackagesUI.csproj

```diff
@@ -33,8 +33,8 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Greg, Version=1.0.6176.18754, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Greg.1.0.6176.18754\lib\net45\Greg.dll</HintPath>
+    <Reference Include="Greg, Version=1.2.7121.21318, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Greg.1.2.7121.21318\lib\net45\Greg.dll</HintPath>
```

Comment:
we should delete this project.

Reply:
Totally, is this the js version pm client?

Reply:
yes, we can remove this at any time.

Reply:
I see, seems it is in the repo but not part of the solution, so little risk